avm/res/operations-management/solution/README.md

```diff
@@ -0,0 +1,311 @@
+# Operations Management Solutions `[Microsoft.OperationsManagement/solutions]`
+
+This module deploys an Operations Management Solution.
+
+## Navigation
+
+- [Resource Types](#Resource-Types)
+- [Usage examples](#Usage-examples)
+- [Parameters](#Parameters)
+- [Outputs](#Outputs)
+- [Cross-referenced modules](#Cross-referenced-modules)
+
+## Resource Types
+
+| Resource Type | API Version |
+| :-- | :-- |
+| `Microsoft.OperationsManagement/solutions` | [2015-11-01-preview](https://learn.microsoft.com/en-us/azure/templates/Microsoft.OperationsManagement/2015-11-01-preview/solutions) |
+
+## Usage examples
+
+The following section provides usage examples for the module, which were used to validate and deploy the module successfully. For a full reference, please review the module's test folder in its repository.
+
+>**Note**: Each example lists all the required parameters first, followed by the rest - each in alphabetical order.
+
+>**Note**: To reference the module, please use the following syntax `br/public:avm/res/operations-management/solution:0.1.0`.
+
+- [Using only defaults](#example-1-using-only-defaults)
+- [Ms](#example-2-ms)
+- [Nonms](#example-3-nonms)
+- [WAF-aligned](#example-4-waf-aligned)
+
+### Example 1: _Using only defaults_
+
+This instance deploys the module with the minimum set of required parameters.
+
+
+<details>
+
+<summary>via Bicep module</summary>
+
+```bicep
+module solution 'br/public:avm/res/operations-management/solution:0.1.0' = {
```

Comment:
Requires a re-run of the readme generation with latest upstream changes. Will render this to show `<version>` instead of `0.1.0` (which we will re-introduce at a later stage)

Reply:
done, see: https://github.com/Azure/bicep-registry-modules/pull/610/commits/dbdd0750c87f60895b077de032ef5bafda689118